test/integration/version_upgrade_test.go

```diff
@@ -79,8 +79,12 @@ func TestRunningBinaryUpgrade(t *testing.T) {
 	// Should be a version from the last 6 months
 	legacyVersion := "v1.6.2"
 	if KicDriver() {
-		// v1.8.0 would be selected, but: https://github.com/kubernetes/minikube/issues/8740
-		legacyVersion = "v1.9.0"
+		if arm64Platform() {
+			legacyVersion = "v1.17.0"
```

Comment:
comment why this version is different for arm 
for example 

# arm was only available since ...

Reply:
done